### PR TITLE
make InstantHandler support Int64

### PR DESCRIPTION
**Pull request by kwon0408**

### CODE REVIEW

**Comment by kwon0408** [comment]
Thank you for the advice.

I chose the option 1, reverted the original `InstantHandler`, added 3 new handlers and a parameter to `Setup()` with default value for the original one.

File: Ark.Tools.Nodatime.Dapper/InstantTickHandler.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
+﻿using Dapper;
+
+using NodaTime;
+using NodaTime.Text;
```
Comment (AndreaCuneo) [inline Ark.Tools.Nodatime.Dapper/InstantTickHandler.cs:4]: I think this is unused now. please remove unsued `using`

**Review by AndreaCuneo — CHANGES_REQUESTED** [review]
please fix compilation errors.

**Comment by AndreaCuneo** [comment]
Hi @kwon0408 

I've a v4.8 release planned this week, would be grand if I can include this too.

Currently it's failing to build, looks a trivial missing `using`. If you can refresh the branch with a fix I'm going to merge it.

**Comment by kwon0408** [comment]
The build was successful after adding 1 `using` line.

**Comment by AndreaCuneo** [comment]
Included in v4.8.0-beta04 and onward.

**Pull request merged**